lib/command/src/Obelisk/Command/Thunk.hs

```diff
@@ -621,6 +623,20 @@ packThunk' noTrail thunkDir = checkThunkDirectory thunkDir >> readThunk thunkDir
       liftIO $ createThunk thunkDir thunkPtr
       pure thunkPtr
 
+setThunk :: MonadObelisk m => String -> FilePath -> m ()
+setThunk = setThunk' False
+
+setThunk' :: MonadObelisk m => Bool -> String -> FilePath -> m ()
+setThunk' noTrail branch thunkDir = do
+  _ <- unpackThunk' noTrail thunkDir
```

Comment:
What if it's already unpacked?

Reply:
Also if this command fails, I think it should leave the user's thunk in *exactly* the same state it was before.